config/locales/zh.yml

```diff
@@ -880,8 +880,8 @@ forms.webauthn_platform_setup.intro_html:
 forms.webauthn_platform_setup.intro_link_text: 了解关于使用不同设备的更多信息。
 forms.webauthn_platform_setup.nickname: 设备昵称
 forms.webauthn_platform_setup.nickname_hint: 这样如果你添加了更多使用人脸或触摸解锁的设备的话，你就能把它们分辨开来。
-forms.webauthn_setup.intro: 使用实体安全密钥为您的 %{app_name} 帐户添加了额外一层安全保障，以防止未经授权的访问。了解有关安全密钥的更多信息
-forms.webauthn_setup.learn_more: Learn more about security keys
+forms.webauthn_setup.intro: 使用实体安全密钥为您的 %{app_name} 帐户添加了额外一层安全保障，以防止未经授权的访问。
+forms.webauthn_setup.learn_more: 了解有关安全密钥的更多信息
```

Comment:
This was translated as one piece of content in two sentences, but they were later split apart into two separate keys.